client/src/pages/FiltersPage/FiltersPage.js

```diff
@@ -70,6 +71,34 @@ export default function FiltersPage() {
     return <div>Loading...</div>;
   }
 
+  function findSortType() {
+    if (sortType == 'alphabetical') {
+      return sortAlphabetically;
+    } else if (sortType == 'distance') {
+      return sortAlphabetically; // replace with 'sortLocation' once location option is configured
+    }
+  }
+
+  function sortAlphabetically(coop1, coop2) {
+    return coop1['coop_name'] > coop2['coop_name'] ? 1 : -1;
+  }
+
+  function searchCoops(coop) {
+    if (searchInput == null) return true;
+    else {
+      for (var k in coop) {
+        if (
+          typeof coop[k] == 'string' &&
+          k != 'hashed_pass' &&
```

Comment:
ahh this is clever hehe, but instead of having this check here can we just have it so we don't send back the hashed password in our GET requests? it's good to not expose the hashed password to the client anyway since we don't need to use it

basically, rather than selecting `*`, select all the fields in `coops` except for ` hashed_pass`. here are the offending locations for your convenience: [1](https://github.com/codebase-berkeley/cofed/blob/main/api/routes/index.js#L58-L59), [2](https://github.com/codebase-berkeley/cofed/blob/main/api/routes/index.js#L79), [3](https://github.com/codebase-berkeley/cofed/blob/main/api/routes/index.js#L31)

Reply:
let's remove this line now since we won't ever have that key anymore